src/Cache_Command.php

```diff
@@ -339,4 +339,36 @@ public function type( $args, $assoc_args ) {
 		WP_CLI::line( $message );
 	}
 
+	/**
+	 * Removes all cache items in a group, if the object cache implementation supports it.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <group>
+	 * : Cache group key.
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     # Clear cache group.
+	 *     $ wp cache clear-group my_group
+	 *
+	 * @subcommand clear-group
+	 */
+	public function clear_group( $args, $assoc_args ) {
+		list( $group ) = $args;
+
+		if ( ! function_exists( 'wp_cache_supports' ) ) {
+			WP_CLI::error( 'Checking cache features is only available in WordPress 6.1 and higher' );
+		}
+
+		if ( ! wp_cache_supports( 'flush_group' ) ) {
+			WP_CLI::error( 'Group flushing is not supported.' );
+		}
+
+		if ( ! wp_cache_flush_group( $group ) ) {
+			WP_CLI::warning( "Cache group '$group' was not be flushed." );
```

Comment:
```suggestion
			WP_CLI::error( "Cache group '$group' was not flushed." );
```

It looks like `wp cache delete` errors:

```
$ wp cache delete foo
Error: The object was not deleted.
```